server/src/unittest/gpos/task/CAutoTaskProxyTest.cpp

```diff
@@ -539,6 +567,57 @@ CAutoTaskProxyTest::EresUnittest_PropagateExecError()
 	return GPOS_OK;
 }
 
+//---------------------------------------------------------------------------
+//	@function:
+//		CAutoTaskProxyTest::EresUnittest_ExecuteError
+//
+//	@doc:
+//		Check if we can see the exception if task and autotask run in same thread
```

Comment:
Also update the intend of this test to check "whether exception propagation is controlled by the `fPropagateException`"